frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_templates/whatsapp_templates.py

```diff
@@ -14,9 +14,7 @@ class WhatsAppTemplates(Document):
     def after_insert(self):
         """Set template code."""
         self.template_name = self.template_name.lower().replace(' ', '_')
-        self.language_code = frappe.db.get_value(
-            "Language", self.language
-        ).replace('-', '_')
+        self.language_code = frappe.db.get_list('Language', filters={'language_name': ['=', self.language]})[0].name.replace('-', '_')
```

Comment:
this change doesn't make sense right? instead of fetching the value you are fetching the list

Reply:
the result is the same in both cases.

Reply:
pl check again
 field is **language_name** of 'Language' doctype which should be compared

Reply:
No. We need the name field of the language doctype. The name field has the language code.